src/main/java/io/fusionauth/load/FusionAuthCreateTenantWorker.java

```diff
@@ -33,20 +33,20 @@
 /**
  * @author Daniel DeGroff
  */
-public class FusionAuthCreateTenantWorker extends BaseWorker {
-  private final FusionAuthClient client;
-
+public class FusionAuthCreateTenantWorker extends FusionAuthBaseWorker {
   private final AtomicInteger counter;
 
   public FusionAuthCreateTenantWorker(FusionAuthClient client, Configuration configuration, AtomicInteger counter) {
-    super(configuration);
-    this.client = client;
+    super(client, configuration);
     this.counter = counter;
   }
 
   @Override
   public boolean execute() {
-    Tenant tenant = new Tenant().with(t -> t.name = "tenant_" + counter.incrementAndGet())
+    int tenantIndex = counter.incrementAndGet();
+    setTenantIndex(tenantIndex);
```

Comment:
There are a lot of places in the workers and base class that have local variables with the same name as class variables. I think it's better to avoid that except in constructors. It would be easy to forget to set a value or use the wrong one. What about
```suggestion
    setTenantIndex(counter.incrementAndGet());
```
That would leave us with just one `tenantIndex` variable.